west.yml

```
@@ -157,7 +157,7 @@ manifest:
       revision: 12019623bbad9eb54fe51066847a7cbd4b4eac57
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: 783c1f78c8e39751fe89d0883c8bce7336f55e94
+      revision: b63b926f0ea64d05a29d3488d615445bf98317a1
```

You should point to the PR here
```suggestion
      revision: pull/15/head
```